### ISSUE TITLE
Batch governance proposal UX improvements

### ISSUE THREAD

**Issue by JonRay15** [vegaprotocol/frontend-monorepo#5862]
## The Chore

A few remaining snags on batch governance

## Simplify Vote / Summary section

Barney raised concerns the view is currently overlying duplicative.... I suggest that we:

1. Replace the "proposals in batch" section at the top with the overall vote so that the view is more aligned to what we do on all other vote types

You can get the overall thresholds from the batch terms at the start of the API.

![image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/ebb4c98a-6f64-4b63-aa66-28daf588564c)

So we would just say:

Batch Proposal: Update market x 5, Update market state x 5 etc
Plus the usual info about how long is left to vote and whether it is currently expected to pass / fail
Then we show that overall vote

![image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/4b9ca806-739e-47c6-be20-184231fe8e08)

Basically we align it to other proposal types

2. Add the link / copy button to the details section

So just add them to the text line here if we can ... that way we dont lose that quite useful quick link to Console

![image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/827387e4-09a2-4c8e-b032-70c45885d18d)
